editor/connections_dialog.h

```diff
@@ -35,16 +35,20 @@
 #include "editor/editor_inspector.h"
 #include "editor/scene_tree_editor.h"
 #include "scene/gui/button.h"
+#include "scene/gui/check_box.h"
 #include "scene/gui/check_button.h"
 #include "scene/gui/dialogs.h"
+#include "scene/gui/label.h"
 #include "scene/gui/line_edit.h"
 #include "scene/gui/menu_button.h"
+#include "scene/gui/option_button.h"
 #include "scene/gui/popup.h"
+#include "scene/gui/popup_menu.h"
+#include "scene/gui/spin_box.h"
 #include "scene/gui/tree.h"
 
-class PopupMenu;
+class EditorNode;
```

Comment:
It's not used.

Reply:
Right, I started this work before #57306 was merged so I had added a few forward declares, seems I didn't remove them all.

Reply:
Fixed, together with a handful of other unneeded `EditorNode` forward declares I had missed.